script.js

```diff
@@ -0,0 +1,55 @@
+const tarefa = document.querySelector('#task')
+const tabela = document.querySelector('#toDo-list')
+const botaoAdicionarTarefa = document.querySelector('#add-task')
+
+function remove() {
+  let remover = document.getElementsByClassName('remove')
+
+  for (elem of remover) {
```

Comment:
isto esta errado, esta forçando adicionar evento em elementos que ja tem o evento de click